components/credentials/src/main/java/org/cloudfoundry/credhub/generators/SignedCertificateGenerator.java

```diff
@@ -93,11 +93,24 @@ private X509Certificate getSignedByIssuer(
 
     final BigInteger certificateSerialNumber = serialNumberGenerator.generate();
 
+    Date nvb = Date.from(now);
+    Data nva = Date.from(now.plus(Duration.ofDays(params.getDuration())));
```

Comment:
This doesn't compile - should be "Date". Please run "gradlew test" after fixing.